examples/vacanza.py

```
@@ -0,0 +1,131 @@
+    except ValueError:
+        print(f"Error: Invalid year '{year_arg}'. Use format YYYY or YYYY-YYYY.")
+        sys.exit(1)
+
+
+def get_supported_categories(country_code):
+    """Get all supported holiday categories for a given country."""
+    try:
+        country_class = holidays.country_holidays(country_code)
+        if hasattr(country_class, "supported_categories"):
+            return country_class.supported_categories
+        return ("PUBLIC",)
```

Ruff (0.15.6)

[warning] Consider moving this statement to an `else` block

(TRY300)